00-Install_and_Setup/check_env.py

```diff
@@ -36,22 +36,20 @@ def check_package(package_name, minimum_version=None, verbose=True):
     return errors
 
 
-pkgs = {'IPython': '5.1',
-        'notebook': '4.2.3',
-        'numpy': '1.7',
-        'scipy': '0.15',
-        'matplotlib': '1.3',
-        'astropy': '1.2.1',
-        'photutils': '0.4',
-        'skimage': '0.12.3',
-        'pandas': '0.18.1',
+pkgs = {'IPython': '7.2',
+        'notebook': '5.7',
+        'numpy': '1.14',
+        'scipy': '1.1',
+        'matplotlib': '2.0',
```

Comment:
still need the 2.2.3 version of matplotlib here

Reply:
Just addressed it.